scripts/tests.e2e.sh

```diff
@@ -102,7 +102,7 @@ rm -rf $snapshots_dir
 killall network.runner || echo
 
 echo "launch local test cluster in the background"
-/tmp/network.runner \
+bin/avalanche-network-runner \
```

Comment:
A minor thing, where there's no real consensus I guess: I try to avoid to generate stuff into the working directory. For example, when moving around (copy) the whole directory (for example when copying to an experimental instance, where we can't use git), it gets copied with it, even if it's in `.gitignore`...

Reply:
I guess I'm of the opposing philosophy. Generally, if a script generates something, it should be in the same working directory. If you wanted to copy the directory cleanly, you could just clone again. If we did want to change this, I think the better idea would be to adjust the build script to accept a custom install location as a command line flag. But default should still be bin.

Reply:
Ok, I went ahead and changed the build script. Now you can build to tmp if you wish, but it won't be default

Reply:
That's very nice, thanks.